gglasso/helper/data_generation.py

```diff
@@ -216,13 +220,15 @@ def sample_covariance_matrix(Sigma, N, seed = None):
     return: sample covariance matrix S
     """
     if seed is not None:
-        np.random.seed(seed)
-    
+        rng = np.random.default_rng(seed)
```

Comment:
Do this unconditionally. `default_rng(None)` will pull random entropy from the OS and will be vastly more likely to give you a distinct PRNG than `np.random.randint(low=11111, high=99999)` will.